src/mmw/js/src/modeling/controls.js

```diff
@@ -57,29 +59,70 @@ var DrawControlView = ControlView.extend({
     }
 });
 
-var LandCoverView = DrawControlView.extend({
-    template: landCoverTmpl,
+var SummaryView = Marionette.ItemView.extend({
+    template: summaryTmpl,
+
+    modelEvents: {
+        'change:thumbValue': 'render'
+    },
 
     templateHelpers: function() {
+        var thumbValue = this.model.get('thumbValue'),
+            label = '',
+            summary = '';
+        if (thumbValue) {
+            label = models.getHumanReadableLabel(thumbValue);
+            summary = models.getHumanReadableSummary(thumbValue);
+        }
         return {
-            label: models.getHumanReadableLabel
+            label: label,
+            summary: summary
         };
+    }
+});
+
+var ModificationsView = DrawControlView.extend({
+    ui: _.defaults({
+        thumb: '.thumb'
+    }, DrawControlView.prototype.ui),
+
+    regions: {
+        summaryRegion: '.summary-region'
+    },
+
+    events: _.defaults({
+        'mouseenter @ui.thumb': 'setThumbValue',
+        'mouseleave @ui.thumb': 'clearThumbValue'
+    }, DrawControlView.prototype.events),
+
+    templateHelpers: {
+        labelFn: models.getHumanReadableLabel
+    },
+
+    setThumbValue: function(event) {
+        this.model.set('thumbValue', $(event.currentTarget).data('value'));
     },
 
+    clearThumbValue: function() {
+        this.model.set('thumbValue', null);
```

Comment:
As discussed, since this is far away from the model, I'd consider making a setter on the model and explaining in a comment what the value is used for.